flavors/standard-EXASOL-all-python-3.10/flavor_base/base_test_build_run/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM {{language_deps}}
+
+RUN mkdir /conf /buckets
+
+COPY --from={{base_test_deps}} /usr /usr
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /lib /lib
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /bin /bin
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /opt /opt
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /etc /etc
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+COPY --from={{base_test_deps}} /env /env
+RUN true # workaround for https://github.com/moby/moby/issues/37965
+
+
+RUN ldconfig
+
+RUN mkdir /exaudfclient /exaudf
+COPY /exaudfclient /exaudfclient
+
+WORKDIR /exaudfclient/
+RUN ["/bin/bash", "-c", "source /env && bash build.sh --config no-tty -c dbg --config python --config test-binaries"]
+RUN cp -r -L bazel-bin/* /exaudf
+RUN rm -r /root/.cache/bazel #Delete bazel cache as it contains java rules with special characters which will cause a problem for Bucketfs
```

Comment:
What I recognised,  we don't remove the cache in build_run

Reply:
Not necessary in `build_run`. The difference between test and release is that the test img (flavor_test_build_run) uses the `base_test_build_run` as base docker image, whereas the release img uses `ubuntu` as base img.
=> In the release img the directory `/home/root` comes from the `ubuntu` img, whereas in the test img it comes from `base_test_build_run`.

Reply:
but for udfclient tests, we might not build the release, we might only build until build_run